lib/l10n/intl_en.arb

```diff
@@ -59,6 +59,8 @@
   "sectionEventsComingUp": "Events coming up",
   "sectionFAQ": "FAQ",
   "sectionGrading": "Grading",
+  "sectionFeedbackToComplete": "Feedback to complete",
```

Comment:
This is more like a nit, but maybe think of a different way to call this. At a glance, the two headers seem exactly the same (the small "to"/"de" doesn't make much of a visual difference).